app/views/requests/index_drop_off.jbuilder

```diff
@@ -0,0 +1,25 @@
+json.array! @requests do |request|
+  json.id       request.id
+  json.user do
+    json.id               request.user.id
+    json.name             "#{request.user.first_name} #{request.user.last_name}"
+    json.profile_picture  request.user.profile_picture
+    if request.user.is_manual
```

Comment:
I am still against having the same information... `is_manual` and `transmission` are the same thing